pages/agent/v3/signed_pipelines.md

```diff
@@ -45,86 +47,117 @@ Additionally, we also sign the following aspects of the build for each job:
 
 ## Enabling signed pipelines on your agents
 
-### 1. Generating a key pair
-Behind the scenes, Signed Pipelines uses [JSON Web Signing (JWS)](https://datatracker.ietf.org/doc/html/rfc7797) to generate its signatures. This means that you'll need to generate a [JSON Web Key Set (JWKS)](https://datatracker.ietf.org/doc/html/rfc7517) to sign and verify your pipelines with, then configure your agents to use those keys.
+You'll need to configure your agents and update pipeline definitions to enable signed pipelines.
+
+### 1. Generate a key pair
+
+Behind the scenes, signed pipelines use [JSON Web Signing (JWS)](https://datatracker.ietf.org/doc/html/rfc7797) to generate signatures. You'll need to generate a [JSON Web Key Set (JWKS)](https://datatracker.ietf.org/doc/html/rfc7517) to sign and verify your pipelines with, then configure your agents to use those keys.
 
-Luckily, the Buildkite Agent has you covered! There's a JWKS generation tool built into the agent, which you can use to generate a key pair for you. To run it, you'll need to [install the agent on your machine](/docs/agent/v3/installation), and then run:
+Luckily, the agent has you covered! A JWKS generation tool is built into the agent, which you can use to generate a key pair for you. To use it, you'll need to [install the agent on your machine](/docs/agent/v3/installation), and then run:
 
 ```bash
 buildkite-agent tool keygen --alg <algorithm> --key-id <key-id>
 ```
 
-replacing `<algorithm>` and `<key-id>` with the algorithm you want to use, and the key ID you want to use. For example, to generate an EdDSA key pair with a key ID of `my-key-id`, you'd run:
+Replacing the following:
+
+- `<algorithm>` with the signing algorithm you want to use.
+- `<key-id>` with the key ID you want to use.
+
+For example, to generate an EdDSA key pair with a key ID of `my-key-id`, you'd run:
 
 ```bash
 buildkite-agent tool keygen --alg EdDSA --key-id my-key-id
 ```
 
-The agent will then generate two JSON Web Key Sets for you, one private and one public, and output them to files in the current directory. You can then use these keys to sign and verify your pipelines, respectively.
+The agent generates two JWKS in your current directory: one private and one public. You can then use these keys to sign and verify your pipelines.
```

Comment:
the reason i didn't use the acronym here is that JWKS isn't plural (it stands for JSON Web Key Set), and we're generating two JWKSes, and it's a difficult word to pluralize 😅

Reply:
perfect!